package.json

```diff
@@ -80,7 +80,8 @@
     "typescript-strict-plugin": "^2.4.4"
   },
   "resolutions": {
-    "rollup": "4.40.1"
+    "rollup": "4.40.1",
+    "socks": ">=2.8.3"
```

Comment:
Where is this dep pulled in? Wondering if we can bump the parent instead of pinning at the resolution level. Wary of adding more of these since I was recently bitten by the rollup pin

Reply:
I'll find the tree for you when I'm back at a laptop. I'm not too worried about the version here since it just forces "2.8.3" or later, so any subsequent updates will still be pulled in.

Reply:
Ah, fair point about the range.